src/ripple/protocol/impl/STPathSet.cpp

```diff
@@ -156,7 +156,6 @@ Json::Value STPath::getJson(JsonOptions) const
         auto const iType = it.getNodeType();
 
         elem[jss::type] = iType;
-        elem[jss::type_hex] = strHex(iType);
```

Comment:
Breaks backward compatibility

Reply:
Does not: these parameters are deprecated in the documentation and marked as "subject to being removed".